sample/src/main/java/com/canhub/cropper/sample/SampleCrop.kt

```diff
@@ -91,154 +93,118 @@ internal class SampleCrop : Fragment() {
+                    maxCropResultWidth = 99999,
+                    activityTitle = "",
+                    activityMenuIconColor = 0,
+                    outputCompressFormat = Bitmap.CompressFormat.JPEG,
+                    outputCompressQuality = 90,
+                    outputRequestWidth = 0,
+                    outputRequestHeight = 0,
+                    outputRequestSizeOptions = CropImageView.RequestSizeOptions.RESIZE_INSIDE,
+                    initialCropWindowRectangle = null,
+                    initialRotation = 0,
+                    allowCounterRotation = false,
+                    flipHorizontally = false,
+                    flipVertically = false,
+                    cropMenuCropButtonTitle = null,
+                    cropMenuCropButtonIcon = 0,
+                    allowRotation = true,
+                    noOutputImage = false,
+                    fixAspectRatio = false,
                 )
-                // Normal Settings
-                setScaleType(CropImageView.ScaleType.FIT_CENTER)
-                setCropShape(CropImageView.CropShape.RECTANGLE)
-                setGuidelines(CropImageView.Guidelines.ON_TOUCH)
-                setAspectRatio(1, 1)
-                setMaxZoom(4)
-                setAutoZoomEnabled(true)
-                setMultiTouchEnabled(true)
-                setCenterMoveEnabled(true)
-                setShowCropOverlay(true)
-                setAllowFlipping(true)
-                setSnapRadius(3f)
-                setTouchRadius(48f)
-                setInitialCropWindowPaddingRatio(0.1f)
-                setBorderLineThickness(3f)
-                setBorderLineColor(Color.argb(170, 255, 255, 255))
-                setBorderCornerThickness(2f)
-                setBorderCornerOffset(5f)
-                setBorderCornerLength(14f)
-                setBorderCornerColor(WHITE)
-                setGuidelinesThickness(1f)
-                setGuidelinesColor(R.color.white)
-                setBackgroundColor(Color.argb(119, 0, 0, 0))
-                setMinCropWindowSize(24, 24)
-                setMinCropResultSize(20, 20)
-                setMaxCropResultSize(99999, 99999)
-                setActivityTitle("")
-                setActivityMenuIconColor(0)
-                setOutputCompressFormat(Bitmap.CompressFormat.JPEG)
-                setOutputCompressQuality(90)
-                setRequestedSize(0, 0)
-                setRequestedSize(0, 0, CropImageView.RequestSizeOptions.RESIZE_INSIDE)
-                setInitialCropWindowRectangle(null)
-                setInitialRotation(0)
-                setAllowCounterRotation(false)
-                setFlipHorizontally(false)
-                setFlipVertically(false)
-                setCropMenuCropButtonTitle(null)
-                setCropMenuCropButtonIcon(0)
-                setAllowRotation(true)
-                setNoOutputImage(false)
-                setFixAspectRatio(false)
-                // Odd Settings
```

Comment:
Where is the new odd settings?

Reply:
Removed, people can look at the type itself to figure out all options

Reply:
Can you please return the odd settings? I use this for testing when people make changes to the library. 

Or if you really want to remove we can add CI tests that check is this values are been setup.